consensus/src/counters.rs

```diff
@@ -843,6 +843,15 @@ pub static UNEXPECTED_PROPOSAL_EXT_COUNT: Lazy<IntCounter> = Lazy::new(|| {
     .unwrap()
 });
 
+/// Count of the number of `DKG` validator transactions received while the feature is disabled.
+pub static UNEXPECTED_DKG_VTXN_COUNT: Lazy<IntCounter> = Lazy::new(|| {
+    register_int_counter!(
+        "aptos_consensus_unexpected_dkg_vtxn_count",
+        "Count of the number of `DKG` validator transactions received while the feature is disabled."
+    )
+    .unwrap()
+});
```

Comment:
how do we want to consume this metric?

Reply:
Same as `UNEXPECTED_PROPOSAL_EXT_COUNT`?

Reply:
still not sure how :)

Reply:
Will have a figure showing error rate, similar to [this](https://aptoslabs.grafana.net/d/consensus/consensus?from=now-1h&to=now&var-Datasource=fHo-R604z&var-BigQuery=axNEitxVz&var-metrics_source=vmagent&var-chain_name=randomnet&var-cluster=All&var-namespace=All&var-kubernetes_pod_name=All&orgId=1&editPanel=140)